KlaviyoSwiftExtension.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name             = "KlaviyoSwiftExtension"
+  s.version          = "2.0.1"
+  s.summary          = "Incorporate Klaviyo's rich push notifications functionality into your iOS applications"
+
+  s.description      = <<-DESC
+                          Incorporate Klaviyo's rich push notifications functionality into your iOS applications. Written in Swift.'
+                       DESC
+
+  s.homepage         = "https://github.com/klaviyo/klaviyo-swift-sdk"
+  s.license          = { :type => "MIT", :file => "LICENSE" }
+  s.author           = { "Mobile @ Klaviyo" => "mobile@klaviyo.com" }
```

Comment:
Is this email valid?

Reply:
I think so yah! We get emails to it sometimes.